src/availability.rs

```diff
@@ -487,6 +487,7 @@ mod test {
             mocks::{mock_transaction, MockHeader, MockPayload, MockTypes},
             setup_test,
         },
+        types::HeightIndexed,
```

Comment:
This isn't used in this module as far as I can tell.

Reply:
Ah this is a `trait`, I see.

Reply:
It's used for the `.height()` method of leaves, blocks, etc